### DISCUSSION TITLE
ddev on live server

### DISCUSSION THREAD

**Discussion by tagdo** [ddev/ddev#5208] [General]
Hi,

in the last time I saw in an agency ddev running on a live server. Company members working on the server, nothing is on their computer, even .ddev is on github, they install the project directly on the server and work with it on browser window with vscode. Does anyone knows how to install it on the server, not local, with all mentioned tools? I installed it, but cannot acces it by the url?! Thanks in advance. Even documentation link.

**Comment by rfay** [comment]
Hi @tagdo - This section of the docs explains how to share a project on your local network or otherwise. https://ddev.readthedocs.io/en/latest/users/topics/sharing/

This isn't the recommended way of using DDEV - It's intended to be set up for local development. But various folks use it for testing, etc.

**Reply by tagdo** [reply]
Thank you so much rfay, I will try it to finalize the installation.
